library/include/ck/library/utility/check_err.hpp

```diff
@@ -171,9 +169,12 @@ check_err(const std::vector<T>& out,
 
     for(std::size_t i = 0; i < ref.size(); ++i)
     {
-        if(out[i] != ref[i])
+        const int64_t out_v = static_cast<int64_t>(out[i]);

```

Comment:
we may use `auto` on left-hand-side to avoid duplicated type name: `int64_t`

Reply:
fixed